control/settings/index.html

```diff
@@ -5,13 +5,15 @@
     <link href="../../../../styles/helper.css" rel="stylesheet">
     <link href="../../../../styles/siteIcons.css" rel="stylesheet">
     <link href="./assets/css/control.settings.css" rel="stylesheet">
+    <link href="./assets/css/bf_base.css" rel="stylesheet">
 
     <!--SDK js-->
     <script src="../../../../scripts/buildfire.min.js"></script>
     <script src="../../../../scripts/jquery/jquery-1.11.2.min.js"></script>
     <script src="../../../../scripts/angular/angular.min.js"></script>
     <script src="../../../../scripts/angular/angular-route.min.js"></script>
     <script src="../../../../scripts/angular/angular-animate.min.js"></script>
+    <script src="../../../../scripts/angular/ui-bootstrap.min.js"></script>
```

Comment:
Is there a specific reason why we are importing this? We have bootstrap inherited from Buildfire itself?